keys.go

```diff
@@ -112,13 +112,20 @@ type KeyVersions struct {
 // KeysActionRequest represents request parameters for a key action
 // API call.
 type KeysActionRequest struct {
-	PlainText           string   `json:"plaintext,omitempty"`
-	AAD                 []string `json:"aad,omitempty"`
-	CipherText          string   `json:"ciphertext,omitempty"`
-	Payload             string   `json:"payload,omitempty"`
-	EncryptedNonce      string   `json:"encryptedNonce,omitempty"`
-	IV                  string   `json:"iv,omitempty"`
-	EncryptionAlgorithm string   `json:"encryptionAlgorithm,omitempty"`
+	PlainText           string      `json:"plaintext,omitempty"`
+	AAD                 []string    `json:"aad,omitempty"`
+	CipherText          string      `json:"ciphertext,omitempty"`
+	Payload             string      `json:"payload,omitempty"`
+	EncryptedNonce      string      `json:"encryptedNonce,omitempty"`
+	IV                  string      `json:"iv,omitempty"`
+	EncryptionAlgorithm string      `json:"encryptionAlgorithm,omitempty"`
+	KeyVersion          *KeyVersion `json:"keyVersion"`
+}
+
+type KeyActionResponse struct {
+	PlainText  string     `json:"plaintext,omitempty"`
+	CipherText string     `json:"ciphertext,omitempty"`
+	KeyVersion KeyVersion `json:"keyVersion"`
```

Comment:
KeysActionRequest has the KeyVersion struct variable of type pointer to KeyVersion. 
But here it is not. Can we keep type of KeyVersion same in both the request & response structure?